bundles/org.openhab.voice.mimictts/src/main/java/org/openhab/voice/mimic/internal/MimicTTSService.java

```diff
@@ -325,16 +302,16 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
         }
     }
 
-    private AudioStream createTemporaryFile(InputStream inputStream, AudioFormat audioFormat) throws TTSException {
-        File mimicDirectory = new File(OpenHAB.getUserDataFolder(), "mimic");
-        mimicDirectory.mkdir();
+    @Override
+    public String getCacheKey(String text, Voice voice, AudioFormat requestedFormat) {
```

Comment:
Interesting. I realize that I have to do something similar in VoiceRSS.

Reply:
The default cache key generator is sufficient for most of the case. But for some configuration particularities (pitch, speed, etc.) it could indeed be extended.

Reply:
Yes, you're right, the default key generation is fine for VoiceRSS.